src/main/java/hw7/entities/MetalAndColors.java

```diff
@@ -0,0 +1,42 @@
+package hw7.entities;
+
+import com.epam.jdi.tools.DataClass;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class MetalAndColors extends DataClass<MetalAndColors> {
+
+    private static final String LINE_SEPARATOR = ", ";
+
+    private int oddSummaryNumber;
+    private int evenSummaryNumber;
+    private List<String> elements;
+    private String color;
+    private String metal;
+    private List<String> vegetables;
+
+    public MetalAndColors() {
```

Comment:
What is the reason of NoArgsConstructor here ??

Reply:
There is such error without NoArgsConstructor:
Error:(9, 49) java: constructor MetalAndColors in class hw7.entities.MetalAndColors cannot be applied to given types;
  required: int,int,java.util.List<java.lang.String>,java.lang.String,java.lang.String,java.util.List<java.lang.String>
  found: no arguments
  reason: actual and formal argument lists differ in length

And I can't use  set() from JdiDataClass which my Class MetalAndColors extends without NoArgConstrutor. 

Added Lombok annotation NoArgsConstructor